uber/utils.py

```diff
@@ -336,26 +340,46 @@ def to_dict(self):
         }
 
     @property
-    def models(self):
-        return self._models_cached
+    def is_empty(self):
+        return not self._targets
 
-    @property
+    @cached_property
```

Comment:
This is true for a single Charge() object, but a new Charge() object will be created for each request. So, if someone leaves their screen idle and then clicks the pay button, that will create a new Charge() object and the total_cost is recalculated for that moment in time.